Yosemite/Yosemite/Model/Mocks/Graphs/ScreenshotsObjectGraph.swift

```diff
@@ -0,0 +1,366 @@
+import Foundation
+import Storage
+
+struct i18n {
+    struct DefaultAccount {
+        static let displayName = NSLocalizedString("My Account", comment: "displayName for the screenshot demo account")
+        static let email = NSLocalizedString("woocommercestore@example.com", comment: "email address for the screenshot demo account")
+        static let username = NSLocalizedString("test account", comment: "username for the screenshot demo account")
+    }
+
+    struct DefaultSite {
+        static let name = NSLocalizedString("Your WooCommerce Store", comment: "Store Name for the screenshot demo account")
+        static let url = NSLocalizedString("example.com", comment: "")
+    }
+}
+
+struct ScreenshotObjectGraph: MockObjectGraph {
+
+     public let userCredentials = Credentials(
```

Comment:
ℹ️ This and all the other `public` properties and methods below do not need to be `public`. 🙂 

```suggestion
    let userCredentials = Credentials(
```

Reply:
Yeah – I had these in place in case we were moving to to a package, but I'll go back and audit visibility when we do that in a future PR :) 

Addressed in c99bc531f92a9b1b4fdbb3ba6e5b9eb5efc108b0